website/docs/cloud-docs/workspaces/settings/deletion.mdx

```diff
@@ -0,0 +1,49 @@
+---
+page_title: Destruction and Deletion - Workspaces - Terraform Cloud
+description: |-
+  Learn about destroying infrastructure and deleting workspaces in Terraform Cloud.
+---
+
+# Destruction and Deletion
+
+Terraform Cloud workspaces have two primary delete actions:
+
+- [Destroy Infrastructure](#destroy-infrastructure) deletes resources managed by the Terraform Cloud workspace by triggering a destroy run.
+- [Delete Workspace](#delete-workspaces) deletes the workspace itself without triggering a destroy run.
+
+In general, you should perform both actions in the above order when destroying a workspace to ensure resource cleanup for all infrastructure managed by a workspace.
+
+## Destroy Infrastructure
+
+Destroy plans delete the infrastructure managed by a workspace. It's recommended that you destroy the infrastructure managed by a workspace prior to deleting the workspace itself, otherwise the infrastructure resources will continue to exist but will become unmanaged.
```

Comment:
Brevity suggestion. Do we need to tell people how to do this? Like should we provide guidance on how to create a destroy plan? Or can we link to that? [I think this is the right link](https://terraform-docs-common-git-notchairmk-cloud-eph-a5c940-hashicorp.vercel.app/terraform/cloud-docs/run/modes-and-options#destroy-mode), but would like confirmation!
```suggestion
Destroy plans delete the infrastructure managed by a workspace. We recommend destroying the infrastructure managed by a workspace _before_ deleting the workspace itself. Otherwise, the unmanaged infrastructure resources continue to exist, and you must go into your infrastructure providers to delete the resources manually.
```

Reply:
For context this was mainly copied over from `index.mdx`. The way to create a create a destroy plan from the UI would be using the `Manually destroy` section on this page (I removed the section from this docs PR but can put it back if we want). That link looks correct, we just don't describe how to do it in the UI anywhere in the docs.

Reply:
On second read I think this makes sense as is. Thank you for updating this!